### DISCUSSION TITLE
What parameters should be entered for C_Cpp.updateChannel?

### DISCUSSION THREAD

**Discussion by CodElixer** [microsoft/vscode-cpptools#9652] [Q&A] [answered]
What parameters should be entered under settings.json > C_Cpp.updateChannel ??
The default "Insiders" parameter is deprecated and throwing an error as: "This setting is deprecated. Pre-release extensions are now available via the Marketplace."

![image](https://user-images.githubusercontent.com/40817467/181246033-dae7cb10-3de7-4b30-bb75-d702e2756d30.png)

So, What should be entered instead of "Insiders"?

**Comment by michelleangela** [comment, marked as answer]
@CodElixer
The setting `C_Cpp.updateChannel` is no longer used by the extension and therefore it is deprecated. If you would like to opt into insiders or pre-release versions of the extension, go to the Extensions view in VS Code, select this extension and then choose the option "Switch to Pre-Release Version" to be on the pre-release channel.

**Reply by valentinmunteanu** [reply]
what is the name of the extension which must be selected?

**Reply by bobbrow** [reply]
![image](https://user-images.githubusercontent.com/12818240/234651669-314f730b-7f11-431e-9856-df4e9bcd3060.png)
